### ISSUE TITLE
NoResponse exception in sender.dialog_list with large numbers

### ISSUE THREAD

**Issue by vhdmsm** [luckydonald/pytg#76]
When i run ```dialog_list``` command with large number as input argument it return noResponse exception but in telegram-cli it returns all of list.
For example:
```dialog_list 1500``` in pytg will return noResponse exception while default_answer_timeout is enough time like 60seconds to get the results. This is an issue with tg?
Thanks.

**Comment by luckydonald** [comment]
Could not reproduce that.
```python
from pytg.sender import Sender
s = Sender("localhost", 4458)
s.dialog_list(10000)
```
Try starting the cli with at least the `--json` flag, does that work?

**Comment by vhdmsm** [comment]
Yes what i said was with ```json``` flag and it's okay with cli, but pytg raises this exception:
```
    dialog_list = sender.dialog_list(1500)
  File "/usr/local/lib/python3.4/site-packages/pytg-0.4.5-py3.4.egg/pytg/sender.py", line 570, in command_alias
  File "/usr/local/lib/python3.4/site-packages/pytg-0.4.5-py3.4.egg/pytg/sender.py", line 272, in execute_function
  File "/usr/local/lib/python3.4/site-packages/pytg-0.4.5-py3.4.egg/pytg/sender.py", line 381, in _do_command
  File "/usr/local/lib/python3.4/site-packages/pytg-0.4.5-py3.4.egg/pytg/sender.py", line 471, in _do_send
pytg.exceptions.NoResponse:  [disable_preview] dialog_list 999
```
P.S: i've modified pytg for supporting channels by myself.

**Comment by luckydonald** [comment]
You issue `dialog_list(1500)` but it just execute that as `999`? Thats strange.

```python
# add this to the first lines in your file
import logging
logging.basicConfig(level=logging.DEBUG)
```
What does the logs print?

Regarding 
> P.S: i've modified pytg for supporting channels by myself.

What part of pytg is not supported?
Also, do you want to contribute your changes and create a pull request?

**Comment by vhdmsm** [comment]
> You wrote dialog_list(1500) but it just execute that as dialog_list(999)? Thats strange.

I was just testing numbers :D it isn't going to be executed with large input. 
> What part of pytg is not supported?
> Also, do you want to contribute your changes and create a pull request?

What i changed before was just changing ```type``` to ```peer_type``` in ```fix_msg_array.py``` file because of channels in ```tg``` that was in test branch which is applied in pytg now(at that time i didn't know that pytg will support test branch too). Anyway i'll change my pytg version to the current version to see it'll work or not.

**Comment by luckydonald** [comment]
Pasting that command into netcat works here...
```sh
mac:~ luckydonald$ nc localhost 4458
[disable_preview] dialog_list 1210
ANSWER 12092
[{"members_num": 0, "admin": {"id": "$01000000000000000000000000000000", "peer_type": "
```

**Comment by vhdmsm** [comment]
How many groups and channels do u have? mine is over 1000+ channels and groups and i took long time to process and nothing shows up.
![screen shot 1395-01-04 at 15 05 02](https://cloud.githubusercontent.com/assets/6222662/13982612/bdcb83a8-f108-11e5-8a18-0327f1aad3e4.png)

**Comment by luckydonald** [comment]
Have you timed, how long it will take?

**Comment by vhdmsm** [comment]
Well i think this is about ```tg``` again, i ran it to the ```tg``` directly and it took long time just like now.

**Comment by luckydonald** [comment]
Hm, I would wrap it with a `try` `except` block then, and retry until you get a result.
